packages/material-ui/src/Toolbar/Toolbar.d.ts

```diff
@@ -3,8 +3,9 @@ import { StandardProps } from '..';
 
 export interface ToolbarProps
   extends StandardProps<React.HTMLAttributes<HTMLDivElement>, ToolbarClassKey> {
```

Comment:
Should this now be OverridableComponent ?

cc @eps1lon

Reply:
That's ok for now. We only applied the new typing to some components. I don't think it's settled that we'll switch at all.